asreview/webapp/src/ProjectComponents/AnalyticsComponents/ProgressDensityChart.js

```diff
@@ -353,10 +356,13 @@ export default function ProgressDensityChart(props) {
                               <Typography variant="subtitle2">
                                 Persistent irrelevant records
                               </Typography>
-                              <Typography variant="body2">
-                                More relevant records might not appear. Refer to
-                                your stopping rules to decide if you want to
-                                continue reviewing.
+                              <Typography
+                                variant="body2"
+                                sx={{ color: "text.secondary" }}
+                              >
+                                Relevant records do not appear. Refer to your
+                                stopping rules to decide if you want to continue
```

Comment:
```suggestion
                                stopping rule to decide if you want to continue
```

Reply:
there is one stopping rule, which can be a collection of heuristics, but it still is one rule, right?